docs/core/app-types.md

```diff
@@ -15,126 +15,5 @@ ms.assetid: 93488a0b-c94c-4ed6-97ea-571bb23a862e
-all the same things as the portable application described in the previous section. For native dependencies, 
-the published output will contain a folder for each [Runtime Identifier (RID)](rid-catalog.md#what-are-rids) that the native dependency supports 
-(and that exists in its NuGet package). 
-
-The below `project.json` sample is showing an example of a portable application with a native dependency. 
-
-```json
-"dependencies": {
-    "Microsoft.NETCore.App": {  
-        "version": "1.0.0",
-        "type": "platform"
-    },
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
-},
-"frameworks": {
-    "netcoreapp1.0": {}
-}
-```
-
-## Self-contained application
-Unlike the portable application, a *self-contained application* does not rely on any shared component to 
-be present on the machine where you want to deploy the application. As its name implies, it means that 
-the entire dependency closure, **including the runtime** is packaged with the application. This makes 
-it larger, but also makes it capable of running on any .NET Core supported platforms with the correct 
-native dependencies, whether it has .NET Core installed or not. This makes it that much 
-easier to deploy to the target machine, since you only deploy your application. 
-
-Since the application carries the runtime within itself, you need to make an explicit choice which platforms your application 
-needs to run on. For instance, if you publish a self-contained application for Windows 10, that same application will 
-not work on macOS or Linux and vice versa. Of course, you can add or remove platforms during development at any given time. 
-
-There are several steps to get to a self-contained application. The first is to remove any `"type": "platform"` properties 
-off of any dependencies you have. Second is to leave the dependency on `Microsoft.NETCore.App` as it will pull in
-all of the rest of things that are needed. 
-
-Finally, you need to add a `runtimes` node in your `project.json` that will list out the 
-[RIDs](rid-catalog.md#what-are-rids) you wish to use. When restoring a project that has the `runtimes` node in it, NuGet 
-will restore the needed runtime for all the RIDs specifies. Then, when you want to publish your application for a given platform,
-you publish it using the `--runtime <RID>` argument to `dotnet publish`. The RID specified in the 
-command invocation **has to be** an RID that is specified in your `project.json`; otherwise, an error is thrown. 
-
-If you want to publish for the RID that represents the operating system you are using the [.NET Core SDK](sdk.md) 
-on, you don't have to specify anything to `dotnet publish`. However, you still have to specify that RID in your 
-`project.json` in order to get a standalone application. 
-
-> An important thing to note is that in the .NET Core Tools Preview 2 timeframe, the self-contained application is being published 
```

Comment:
I wonder if this can be rephrased -- I'm not sure how many developers will realize that the tools and library versioning are not in sync, or really know what .NET Core Tools Preview 2 means. And this will have to be revised when tooling changes anyway, so maybe it's ok to just say currently, the self-contained application is published from...